Gordon360/Controllers/RecIMControllers/LeagueController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+
+namespace Gordon360.Controllers.RecIMControllers
+{
+    [Route("api/recim")]
```

Comment:
It is standard practice to use the name of the controller as a route pre-fix for all routes:

```suggestion
    [Route("api/recim/[controller]")]
```